bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java

```diff
@@ -260,7 +262,11 @@ public ItemStack getItemStack(Player player, ShopCache shopCache) {
             String buyStatus;
 
             if (isPermanent() && shopCache.hasCachedItem(this) && shopCache.getCachedItem(this).getTier() == getContentTiers().size()) {
-                buyStatus = getMsg(player, Messages.SHOP_LORE_STATUS_MAXED);
+                if (!(CraftItemStack.asNMSCopy(i).getItem() instanceof ItemArmor)){
```

Comment:
this must be done through **BedWars.nms** https://github.com/andrei1058/BedWars1058/blob/master/bedwars-api/src/main/java/com/andrei1058/bedwars/api/server/VersionSupport.java#L106

Reply:
done